tests/unit/mathematics/linalg/operations/Eigen3_operations_unittest.cc

```diff
@@ -465,6 +465,63 @@ TEST(LinalgBackendEigen, SGMatrix_cross_entropy)
 	EXPECT_NEAR(ref, result, 1e-15);
 }
 
+TEST(LinalgBackendEigen, SGMatrix_pinvh)
+{
+	float64_t A_data[] = {2.0, -1.0, 0.0,
+												-1.0, 2.0, -1.0,
```

Comment:
the formatting here is a bit all over the place ;)

Reply:
yes yes :) will fix it